std_msgs/Header header
ObjectState[] states